src/components/admin-table/admin-table.js

```diff
@@ -276,14 +272,11 @@ class AdminTable extends React.Component {
   };
 
   handleGridRowsUpdated = ({ fromRow, toRow, updated }) => {
-    console.log('handleGridRowsUpdate', fromRow, toRow, updated); // eslint-disable-line
+    console.log('handleGridRowsUpdate', fromRow, toRow, updated);
     const rows = this.state.rows.slice();
     const newRows = this.state.newRows.slice();
     const updatedRows = this.state.updatedRows.slice();
 
-    // shift enumerated fields to lowercase
```

Comment:
This was a fix to a bug Ryan reported.  Did you remove this code or is this just an artifact of the merge?